src/Meilisearch/Extensions/HttpExtensions.cs

```diff
@@ -81,13 +83,39 @@ internal static void AddDefaultUserAgent(this HttpClient client)
             client.DefaultRequestHeaders.Add("User-Agent", version.GetQualifiedVersion());
         }
 
-        private static StringContent PrepareJsonPayload<T>(T body, JsonSerializerOptions options = null)
+        private static StringContent PrepareJsonPayload<T>(T body, JsonSerializerOptions? options = null)
         {
             options = options ?? Constants.JsonSerializerOptionsWriteNulls;
             var payload = new StringContent(JsonSerializer.Serialize(body, options), Encoding.UTF8, "application/json");
             payload.Headers.ContentType.CharSet = string.Empty;
 
             return payload;
         }
+
+        private static Task<HttpResponseMessage> PatchAsync(this HttpClient client, string? requestUri, HttpContent content, CancellationToken cancellationToken)
+        {
+            var uri = string.IsNullOrEmpty(requestUri) ? null : new Uri(requestUri, UriKind.RelativeOrAbsolute);
+            return client.PatchAsync(uri, content, cancellationToken);
+        }
```

Comment:
Couple of things:

- is there any case in which the `requestUri` will not be sent?
- since this is an internal method, it sounds redundant to validate something we know should be sent (I mean, the end-user has no power over the input in this case).
- if you want to keep the validation you should remove the version with `Uri`  (`PatchAsJsonAsync<TValue>(this HttpClient client, Uri? requestUri, TValue `)

```suggestion
        private static Task<HttpResponseMessage> PatchAsync(this HttpClient client, string requestUri, HttpContent content, CancellationToken cancellationToken)
        {
            return client.PatchAsync(requestUri, content, cancellationToken);
        }
```